stdlib/public/core/DebuggerSupport.swift

```diff
@@ -318,25 +242,20 @@ public enum _DebuggerSupport {
       indent: 0,
       maxDepth: maxItemCounter,
       isRoot: true,
-      parentCollectionStatus: .NotACollection,
+      parentCollectionStatus: .notACollection,
       refsAlreadySeen: &refs,
       maxItemCounter: &maxItemCounter,
-      targetStream: &targetStream)
+      target: &target)
 
-    return targetStream
+    return target
   }
 }
 
-@inline(never)
-public
-func _stringForPrintObject(_ value: Any) -> String {
+public func _stringForPrintObject(_ value: Any) -> String {
   return _DebuggerSupport.stringForPrintObject(value)
 }
 
-@inline(never)
-public
-func _debuggerTestingCheckExpect(_ checked_value: String,
-                                 _ expected_value: String) {}
+public func _debuggerTestingCheckExpect(_: String, _: String) { }
```

Comment:
Seems like this global symbol could be eliminated and users could call into the `_DebuggerSupport` version directly.

Reply:
I think the retain count getters below could be moved into `_DebuggerSupport`, too, so that `stringForPrintObject` isn't the only definition there.

Reply:
Yes, I think there's no reason to keep this global.